addons/csw/functions/fnc_assemble_deployWeapon.sqf

```diff
@@ -4,25 +4,27 @@
  * Deploys the current CSW
  *
  * Arguments:
- * 0: Unit <OBJECT>
+ * 0: Target <OBJECT>
+ * 1: Unit <OBJECT>
+ * 2: Args <ANY>
+ * 3: Action Data <ARRAY>
  *
  * Return Value:
  * None
  *
  * Example:
- * [player] call ace_csw_fnc_assemble_deployWeapon
+ * [cursorObject, player] call ace_csw_fnc_assemble_deployWeapon
  *
  * Public: No
  */
 
 [{
-    params ["_tripod", "_player", "", "_carryWeaponClassname"];
-    if (isNil "_carryWeaponClassname") then { _carryWeaponClassname = secondaryWeapon _player };
+    params ["_tripod", "_player"];
+
+    private _carryWeaponClassname = secondaryWeapon _player;
```

Comment:
I can't find anything about `_carryWeaponClassname` being passed here, which is why I removed it as a parameter.